dist_example.py

```diff
@@ -2,135 +2,75 @@
 from types import SimpleNamespace
```

Comment:
I'd add a file, maybe `DEV.md`, indicating the steps you take to run this example. Do you install any packages or activate the `adadamp.yaml` environment? What input files are required, and what output files are written to disk? Does a Dask cluster need to be running in the background?